lib/cocoapods/validator.rb

```diff
@@ -485,7 +486,6 @@ def add_app_project_import
       app_target = app_project.targets.first
       pod_target = @installer.pod_targets.find { |pt| pt.pod_name == spec.root.name }
       Pod::Generator::AppTargetHelper.add_app_project_import(app_project, app_target, pod_target, consumer.platform_name, use_frameworks)
-      Pod::Generator::AppTargetHelper.add_swift_version(app_target, swift_version)
```

Comment:
this is actually better to happen _earlier_. We often validated targets and such and set the `SWIFT_VERSION` *afterwards*. All existing tests pass.